client/index.css

```diff
@@ -199,3 +199,112 @@ ul {
 .chat ul li.typing div {
     animation: flash 2s infinite;
 }
+#settings {
+    width: 345px;
+    height: auto;
```

Comment:
I think "width" is needed because of "margin: auto", but "height" is not needed.